challenges/objects-arrays.js

```diff
@@ -89,34 +130,37 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+zooAnimals.forEach(element => animalNames.push(`${element.animal_name} ${element.scientific_name} ${element.last_name}`));
 console.log(animalNames);
 
-/* Request 2: .map()    
-
+/* Request 2: .map()    LOWER CASE
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-
 */
-
-const lowerCase = [];
+const lowerCase = zooAnimals.map(function nameToLowCase (animal) {
+  return (animal.animal_name.toLowerCase())});
 console.log(lowerCase); 
 
-/* Request 3: .filter() 
+//ARROW
+// const lowerCase = zooAnimals.map((animal) => animal.animal_name.toLowerCase());
 
+/* Request 3: .filter() POPULATION LESS THAN 5
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
-
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter(function lessThanFive (runningTotal, animal) {
+return runningTotal += animal.population < 5;
```

Comment:
looks like you've got a reduce-esque thing going on. could just say `const largerPopulation = zooAnimals.filter(function(animal) {
  return animal.population < 5;
}); `